RestAPI_Automation/Requests/APIHelper.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Newtonsoft.Json;
+using RestSharp;
+
+namespace RestAPI_Automation.Requests;
+
+public class APIHelper<T>
+{
+    public string baseURL = "https://reqres.in/ ";
+
+    // concatenating base url and endpoint
+    public string SetUrl(string endpoint)
+    {
+        string url = Path.Combine(baseURL, endpoint);
+        return url;
+    }
+
+    // method returns RestClient value
+    public RestClient RestClient()
+    {
+        var client = new RestClient(baseURL);
+        return client;
```

Comment:
do we need each time create client? Or we can create static client?

Reply:
corrected